content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -610,6 +610,31 @@ controller is tracking a Job using Pod finalizers by checking if the Job has the
 annotation `batch.kubernetes.io/job-tracking`. You should **not** manually add
 or remove this annotation from Jobs.
 
+### Mutable Scheduling Directives
+
+{{< feature-state for_k8s_version="v1.23" state="beta" >}}
+
+{{< note >}}
+In order to use this behavior, you must enable the `JobMutableNodeSchedulingDirectives`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+on the [API server](/docs/reference/command-line-tools-reference/kube-apiserver/).
+It is enabled by default.
+{{< /note >}}
+
+In most cases a parallel job will want the pods to run within specific constraints, 
+like all in the same zone, or all either on GPU model x or y, but not a mix of both.
+
+The Suspend flag is the first step towards achieving those semantics. Suspend allows a 
```

Comment:
I think the documentation for suspend is not very explicit in telling that you can create jobs in a suspended state. Can you fix that?

Reply:
I added a sentence at the beginning, you think it is enough?